src/main/java/com/azure/cosmos/examples/changefeed/SampleConfigurations.java

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.cosmos.examples.changefeed;
+
+import com.google.common.base.Strings;
+import org.apache.commons.lang3.StringUtils;
+
+/**
+ * Contains the configurations for tests.
+ * <p>
+ * For running tests, you can pass a customized endpoint configuration in one of the following
+ * ways:
+ * <ul>
+ * <li>-DACCOUNT_KEY="[your-key]" -ACCOUNT_HOST="[your-endpoint]" as JVM
+ * command-line option.</li>
+ * <li>You can set ACCOUNT_KEY and ACCOUNT_HOST as environment variables.</li>
+ * </ul>
+ * <p>
+ * If none of the above is set, emulator endpoint will be used.
+ */
+public final class SampleConfigurations {
+    // REPLACE MASTER_KEY and HOST with values from your Azure Cosmos DB account.
+    // The default values are credentials of the local emulator, which are not used in any production environment.
```

Comment:
local emulator -> Cosmos Public Emulator which can be downloaded from https://aka.ms/cosmosdb-emulator